dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/api/SwordIT.java

```diff
@@ -47,6 +48,10 @@
     private static final String rootDataverseAlias = "root";
     private static String apiTokenSuperuser;
 
+    @Inject
+    SwordConfigurationFactory swordConfigurationFactory;
```

Comment:
This won't be injected - It's junit test class without any additional configuration.
As a workaround I propose to get rid of this field and in test method use string `2.0` instead of value from factory